NAMESPACE

```diff
@@ -6,6 +6,7 @@ export(get_var_stats)
 export(send_notification)
 importFrom(dplyr,"%>%")
 importFrom(dplyr,arrange)
+importFrom(dplyr,recode)
 importFrom(rlang,parse_exprs)
 importFrom(tibble,column_to_rownames)
 importFrom(tibble,rownames_to_column)
```

Comment:
This popped up when I did the roxygen command, I guess it should have been there before, but it didn't cause any errors?